.travis.yml

```diff
@@ -18,8 +18,9 @@ install:
   - hack/amptools make -f Makefile.refactor.make build
   - make fmt check install
   - make TAG=local build-image
-  - amp platform pull --local
-  - amp platform start --local
+  - amp platform pull
+  - amp platform start all --local
+  - sleep 5
```

Comment:
was not enough on my laptop, the tests failed (elasticsearch took time to be available, amp-log-worker had a failed task, log tests failed). An arbitrary number of second is maybe not the best solution, the retry mechanism should be implemented in the tests instead.